internal/util/fileop.go

```diff
@@ -234,3 +233,67 @@ func resolvePath(path string, baseDir string, forceUnix bool) string {
 
 	return path
 }
+
+func handleGlobFileOperation(sourceGlob string, dest string, operation func(string, string) error) error {
+	sourcePaths, err := prepForGlobs(sourceGlob, dest)
+	if err != nil {
+		return fmt.Errorf("Failed to prep globs '%s' for a move: '%w'.", sourceGlob, err)
+	}
+
+	var errs error
+
+	for _, sourcePath := range sourcePaths {
+		if sourcePath == dest {
+			continue
+		}
+
+		err = operation(sourcePath, dest)
+		if err != nil {
+			errs = errors.Join(errs, err)
+		}
+	}
+
+	return errs
+}
+
+func handleGlobRemove(path string) error {
+	paths, err := filepath.Glob(path)
+	if err != nil {
+		return fmt.Errorf("Failed to resolve globs '%s': '%w'.", path, err)
+	}
+
+	var errs error
+
+	for _, path := range paths {
+		err = RemoveDirent(path)
+		if err != nil {
+			errs = errors.Join(errs, err)
+		}
+	}
+
+	return errs
+}
+
+func prepForGlobs(globPath string, destPath string) ([]string, error) {
+	sourcePaths, err := filepath.Glob(globPath)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to resolve globs '%s': '%w'.", globPath, err)
+	}
+
+	if len(sourcePaths) == 0 {
+		return nil, fmt.Errorf("Unable to find source path: '%s'.", globPath)
+	}
+
+	if len(sourcePaths) > 1 {
+		err = MkDir(destPath)
+		if err != nil {
+			return nil, fmt.Errorf("Failed to create dest dir '%s': '%w'.", destPath, err)
+		}
+
+		if !IsDir(destPath) {
```

Comment:
How could this happen given the line above?
Doesn't MkDir() error if a non-dir is there?
And if it doesn't we can just beef-up MkDir().